Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -203,10 +206,15 @@ open class NavigationMapView: UIView {
      updated to fit all routes.
      - parameter animated: `true` to asynchronously animate the camera, or `false` to instantaneously
      zoom and pan the map.
+     - parameter duration: Duration of the animation (in seconds). In case if `animated` parameter
+     is set to `false` this value is ignored.
+     - parameter completion: A completion handler that will be called once routes presentation completes.
      */
     public func showcase(_ routes: [Route],
                          routesPresentationStyle: RoutesPresentationStyle = .all(),
-                         animated: Bool = false) {
+                         animated: Bool = false,
+                         duration: TimeInterval = 1.0,
```

Comment:
Adding two new parameters to this public method shouldn't be a problem I guess.

Reply:
Remember to mention this renamed method in the changelog.